GeneralsMD/Code/GameEngine/Include/Common/BezierSegment.h

```diff
@@ -31,7 +31,7 @@
 #ifndef __BEZIERSEGMENT_H__
 #define __BEZIERSEGMENT_H__
 
-#include <d3dx9math.h>
```

Comment:
D3DX9 math should be usable instead of D3DX8 math to my knowledge.
This needs to be confirmed or debunked.

Until then, please put an ifdef for _WIN32 around the DX8 variant.
(And leave d3dx9 in for cross-platform builds).